.cicd/pipeline.yml

```diff
@@ -3,78 +3,100 @@ env:
   TEST_TIMEOUT: 60
   TIMEOUT: 120
   VERBOSE: true
+  DEBUG: true
```

Comment:
This whole block of variables is unnecessary complexity.

Reply:
Agreed, removing